### ISSUE TITLE
[BUG]flag provided but not defined: -test.timeout

### ISSUE THREAD

**Issue by ccb1900** [douyu/jupiter#132]
## Issue Description

Type: *bug report*

### Describe what happened
write a test file like below:

```golang
package xecho

import (
	"github.com/douyu/jupiter"
	"github.com/labstack/echo/v4"
	"testing"
)

func TestServer_Serve(t *testing.T) {
	t.Log("test over")
	var app jupiter.Application

	_ = app.Startup()
	_ = app.Serve(startServer())
	_ = app.Run()
}

func startServer() *Server  {
	serve := DefaultConfig().Build()

	serve.GET("/", func(context echo.Context) error {
		return context.JSON(200,"test echo")
	})

	return serve
}

```

得到

```
flag provided but not defined: -test.timeout
Usage of xecho.test:
  -config string
        --config
  -help
        --help, show help information
  -version
        --version, print version
  -watch
        --watch, watch config change event
exit status 2
FAIL    github.com/douyu/jupiter/pkg/server/xecho       2.021s

```
### Describe what you expected to happen

正常运行

### How to reproduce it (as minimally and precisely as possible)

1. 首先写一个测试文件，上面的代码可以直接使用 pkg/server/xecho/
2. go test

#### What version of Go are you using (`go version`)?

<pre>
$ go version
go version go1.14.5 darwin/amd64
</pre>

#### What operating system and processor architecture are you using (`go env`)?

<details><summary><code>go env</code> Output</summary><br><pre>
$ go env
GO111MODULE="on"
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/ccb/Library/Caches/go-build"
GOENV="/Users/ccb/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/ccb/go"
GOPRIVATE=""
GOPROXY="https://goproxy.cn,direct"
GOROOT="/usr/local/Cellar/go/1.14.5/libexec"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/Cellar/go/1.14.5/libexec/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/ccb/Desktop/github/jupiter/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/kv/nz0sggtx5qn8st6p7fz0fhpm0000gn/T/go-build410110268=/tmp/go-build -gno-record-gcc-switches -fno-common"

</pre></details>

### Anything else we need to know?

我查了一下 好像是 flag.Parse  执行的时间不对。

**Comment by mlboy** [comment]
fixes #133

**Comment by bigrocs** [comment]
怎么处理 没有看明白
